### PR TITLE
Bch submissions changes

### PR DESCRIPTION
**Pull request by tahmeedshah**
DEVDESK-511
DEVDESK-513
DEVDESK-514
DEVDESK-515

BCH- Submissions- "Cosmetic" changes in Edit and View

### CODE REVIEW

**Review by blaisefonseca — CHANGES_REQUESTED** [review]
can we not use auto format please, can you revert the formatting changes?

**Review by blaisefonseca — APPROVED** [review]
ignore my previous comment

File: app/views/forms/edit/edit-submission.html [hunk 1]
```diff
@@ -36,23 +36,28 @@
...
-                                                caption="Relevant Organization(s) " required>
-                                                <div document-selector ng-disabled="document.government" question="organization" type="radio"
-                                                    query="organizationQuery" ng-change="onNotificationSelected()"
-                                                    skip-government="true" ng-model="document.organization" allow-new-schema="organization">
+                                                caption="Relevant organization(s) " required>
+                                                <div document-selector ng-disabled="document.government"
+                                                    question="organization" type="radio" query="organizationQuery"
+                                                    ng-change="onNotificationSelected()" skip-government="true"
+                                                    ng-model="document.organization" allow-new-schema="organization">
+                                                </div>
```
Comment (blaisefonseca) [inline app/views/forms/edit/edit-submission.html:46]:
```suggestion
                                                caption="Relevant organization(s) " required>
                                                <div document-selector ng-disabled="document.government"
                                                    question="organizations" type="radio" query="organizationQuery"
                                                    ng-change="onNotificationSelected()" skip-government="true"
                                                    ng-model="document.organizations" allow-new-schema="organization">
                                                </div>
```

Comment (blaisefonseca) [reply app/views/forms/edit/edit-submission.html:46]: @tahmeedshah  the field has been changed from `organization` to `organizations`, can you please update edit and view forms

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**